buildSrc/src/test/java/org/opensearch/gradle/DistributionDownloadPluginTests.java

```diff
@@ -96,6 +96,11 @@ public void testTypeDefault() {
         assertEquals(distro.getType(), Type.ARCHIVE);
     }
 
+    public void testCustomDistributionUrl() {
```

Comment:
Could you add another test which sets the customDistributionUrl and make sure DistributionDownloadPlugin downloads the artifact?

Reply:
Sure @saratvemulapalli I will add another test for this scenario.